Library/Homebrew/download_strategy.rb

```diff
@@ -406,7 +406,12 @@ def fetch(timeout: nil)
       else
         begin
           _fetch(url: url, resolved_url: resolved_url, timeout: end_time&.remaining!)
-        rescue ErrorDuringExecution
+        rescue ErrorDuringExecution => e
+          if e.status.exitstatus == 22 && resolved_url != url && is_redirection
+            ohai "Downloading from #{url}"
+            resolved_url = url
```

Comment:
Can you explain why this fixes things? It feels like a strange fix to override our internal state about what the original URL was like this. Thanks!

Reply:
I don't quite remember... It's technically not needed though, so I'll remove it.